src/tests/utils.tsx

```diff
@@ -9,6 +9,9 @@ import { ThemeProvider } from 'styled-components/native'
 
 import { computedTheme } from './computedTheme'
 
+// TODO(PC-20887): Investigate how to avoid timeouts in CI without increasing default timeout
+export const WAIT_FOR_TIMEOUT_IN_MS = 10000
```

Comment:
je trouve ça plus lisible d'écrire les gros nombres avec des underscores
```suggestion
export const WAIT_FOR_TIMEOUT_IN_MS = 10_000
```